tests/sniffTest

```diff
@@ -68,15 +68,13 @@ if [[ $rc != 0 || -s err ]]; then
 	echo Output: $(cat out)
 	exit 1
 fi
-$RUNC kill c1
-rm -rf /run/opencontainer/containers/c1
```

Comment:
Or could we do some sort of `while` loop while the state hasn't changed? I don't like magic sleep constants.